c2corg_ui/static/partials/feedtable.html

```diff
@@ -0,0 +1,66 @@
+<div ng-show="wfeedCtrl.documents.length > 0"
+  infinite-scroll="wfeedCtrl.getDocumentsFromWhatsnewFeed()"
+  infinite-scroll-immediate-check="false"
+  infinite-scroll-distance="0.7"
+  infinite-scroll-disabled="wfeedCtrl.busy || wfeedCtrl.error || wfeedCtrl.feedEnd || wfeedCtrl.noFeed">
+
+  <div class="table-responsive">
+    <table class="text-center feedtable table table-condensed table-striped table-hover">
+
+      <p class="text-success text-center" ng-if="wfeedCtrl.userId"
+         ng-init="user = wfeedCtrl.userId;" ng-model="amount = wfeedCtrl.feedLength;">
+        This feed for the user {{user}} contains {{amount}} changes.
+      </p>
+      <p class="text-success text-center" ng-if="!wfeedCtrl.userId"
+         ng-model="amount = wfeedCtrl.feedLength;">
+        This feed contains {{amount}} changes.
+      </p>
+
+      <tr>
+        <th class="text-center" translate>title</th>
+        <th class="text-center" translate>type</th>
+        <th class="text-center" translate>written_at</th>
+        <th class="text-center" translate>quality</th>
+        <th class="text-center" translate>username</th>
+        <th class="text-center" translate>comment</th>
+      </tr>
+
+      <tr class="feed-list"
+              ng-repeat="doc in wfeedCtrl.documents track by doc.version_id"
+          ng-init="type = wfeedCtrl.documentType(doc.document.type); console.log(type);" >
+
+        <td class="text-left">
+          <a href="{{wfeedCtrl.getDocHref(doc)}}">{{::doc['document']['title']}}</a> ({{::doc['lang']}})
+          <span ng-if="doc['document']['version'] > 1 && (doc['document']['type'] !== 'i' && doc['document']['type'] !== 'o')">
+            (<a ng-href="{{wfeedCtrl.getVersionHref(doc)}}">previous version</a> | diff)
+          </span>
+        </td>
+
+        <td>{{::type}}</td>
+
+        <td data-original-title="999" data-container="body" tooltip-append-to-body="true" tooltip-placement="top" uib-tooltip="{{::doc['written_at']}}">
+          {{::doc['written_at'] | amUtc | amLocal | amDateFormat:'L LTS'}}
+        </td>
+
+        <td ng-if="::doc['document']['quality']" class="quality-{{::doc['document']['quality']}}" tooltip-append-to-body="true"
+            uib-tooltip="{{'quality'| translate}} : {{::doc['document']['quality']}}">
+        </td>
+
+        <td data-original-title="999" data-container="body" tooltip-append-to-body="true" tooltip-placement="top"
+            uib-tooltip="user_id: {{::doc['user']['user_id']}} username: {{::doc['user']['username']}} ">
```

Comment:
We don't need this (user id and username). All need info are provided with ``doc['user']['name']``.